odkx-src/sync-endpoint-user-instructions.rst

```diff
@@ -44,67 +67,42 @@ Creating groups
 .. _sync-endpoint-ldap-assign:
 
 Assigning users to groups
-"""""""""""""""""""""""""
+-------------------------
 
   1. Click: :guilabel:`login` on the right and login as *admin*.
   2. Expand the tree view on the right until you see :guilabel:`ou=default_prefix`, then expand :guilabel:`ou=default_prefix`.
   3. This list is all the groups under *ou=default_prefix*.
-  4. Click on the group that you want to assign users to.
-  5. A few groups are created when the LDAP server is brought up, refer to :doc:`data-permission-filters` for descriptions of these groups.
-  6. If the :guilabel:`memberUid` section is not present:
-
-      a. Choose :guilabel:`Add new attribute`.
-      b. Choose :guilabel:`memberUid` from the dropdown, then enter :guilabel:`uid` of the user you want to assign.
-      c. Click :guilabel:`Update Object` at the bottom to update.
+  4. Click on the group that you want to assign users to. In this section, click on :guilabel:`gidNumber=503`, which is the group ID that corresponds to *default_prefix_synchronize_tables*. Groups correspond to the access permissions available to a certain user.
 
-  7. If the :guilabel:`memberUid` section is present,
-
-    a. Navigate to the :guilabel:`memberUid` section.
-    b. Click modify group members to manage members.
-    
-.. _sync-endpoint-setup-create-user:
-
-Creating a Sample User
-----------------------
-
-  | 1. Start by logging into the ldap-service. Copy the login below.
-  |   - login DN: :guilabel:`cn=admin,dc=example,dc=org`
-  |   - password: :guilabel:`admin`
-
-      .. image:: /img/setup-create-user/setup-user1.png
-        :width: 600
-
-  2. Click the :guilabel:`+` sign next to **dc=example, dc=org** to expand it. Within the unfolded menu, in the **ou=people** section, click on :guilabel:`Create a child entry` (new person).
-
-    .. image:: /img/setup-create-user/setup-user2.png
+    .. image:: /img/setup-create-user/setup-user5.png
       :width: 600
 
-  3. Then, select the :guilabel:`Generic: User Account` template.
+  5. A few groups are created when the LDAP server is brought up, refer to :doc:`data-permission-filters` for descriptions of these groups.
 
-    .. image:: /img/setup-create-user/setup-user3.png
-      :width: 600
+   .. note::
 
-  4. Fill out information for the new user and “create object.” Assign it to the *default_prefix_synchronize_tables* group. Will need to commit (confirm) that you want to create this entry on the next screen.
+     A user needs to be assigned one of the roles in addition to any other group of your choosing. These roles are available as groups 500 (Row owner), 501 (Group read only), 502 (Group modify), 503 (Group privileged).
```

Comment:
This is incorrect. 500 is role SITE_ADMIN, 501 is ADMINISTER_TABLES, 502 is SUPER_USER_TABLES, 503 is SYNCHRONIZE_TABLES